addons/mail/static/src/models/follower/follower.js

```diff
@@ -158,6 +158,10 @@ function factory(dependencies) {
                     args: [[this.followedThread.id]],
                     kwargs,
                 }));
+                this.env.services['notification'].notify({
+                    type: 'success',
+                    message: this.env._t("The subscription preferences were successfully applied."),
```

Comment:
The PR is targeting stable so you'll need to add the new translation in the `.pot` file.

Reply:
Added

Reply:
@seb-odoo . Ping